x-pack/plugins/enterprise_search/public/applications/app_search/components/relevance_tuning/relevance_tuning_logic.test.ts

```diff
@@ -586,10 +586,9 @@ describe('RelevanceTuningLogic', () => {
         confirmSpy.mockImplementation(() => false);
 
         RelevanceTuningLogic.actions.resetSearchSettings();
+        await nextTick();
```

Comment:
This was unrelated to the current changes, but I added since I was using this test as a reference.

Without this, it would return a false positive always.

Reply:
++ for the expect cleanup below too!